src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/copy_data_workspaces.cljs

```diff
@@ -15,35 +16,48 @@
   {:render
    (fn [{:keys [props]}]
      [:div {:style {:margin "1em"}}
-      (let [attribute-keys (apply union (map (fn [e] (set (keys (e "attributes")))) (:workspaces props)))]
-        [table/Table
-         {:empty-message "There are no workspaces to display."
-          :columns (concat
-                     [{:header "Google Project" :starting-width 150}
-                      {:header "Name" :starting-width 150
-                       :as-text #(get-in % ["workspace" "name"]) :sort-by :text
-                       :content-renderer
-                       (fn [ws]
-                         (style/create-link {:text (get-in ws ["workspace" "name"])
-                                             :onClick #((:onWorkspaceSelected props) ws)}))}
-                      {:header "Created By" :starting-width 200}
-                      (table/date-column {})
-                      {:header "Access Level" :starting-width 100}]
-                     (map (fn [k] {:header k :starting-width 100}) attribute-keys))
-          :data (:workspaces props)
-          :->row (fn [ws]
-                   (concat
-                    [(get-in ws ["workspace" "namespace"])
-                     ws
-                     (get-in ws ["workspace" "createdBy"])
-                     (get-in ws ["workspace" "createdDate"])
-                     (ws "accessLevel")]
-                    (map (fn [k] (get-in ws ["attributes" k])) attribute-keys)))}])])})
+      [table/Table
+       {:empty-message "There are no workspaces to display."
+        :columns [{:header "Google Project" :starting-width 150}
+                  {:header "Name" :starting-width 150
+                   :as-text #(get-in % ["workspace" "name"]) :sort-by :text
+                   :content-renderer
+                   (fn [ws]
+                     (style/create-link {:text (get-in ws ["workspace" "name"])
+                                         :onClick #((:onWorkspaceSelected props) ws)}))}
+                  {:header "Created By" :starting-width 200}
+                  (table/date-column {})
+                  {:header "Access Level" :starting-width 106}
+                  {:header "Realm" :starting-width 156}]
+        :toolbar (fn [built-in]
+                   [:div {}
+                    [:div {:style {:float "left"}} built-in]
+                    (let [num (:num-filtered props)]
+                      (when (pos? num)
+                        [:div {:style {:float "left" :margin "7px 0 0 1em"}}
+                         (str
+                           (:num-filtered props)
+                           " workspace" (when (> num 1) "s")
+                           " unavailable due to conflicting realms")]))
```

Comment:
I doubt any user would understand this. I know we need PO verbiage, but how about something like "{X} workspace(s) unavailable because they contain protected data"?